sympy/external/tests/test_autowrap.py

```diff
@@ -99,6 +98,44 @@ def runtest_ufuncify(language, backend):
     numpy.testing.assert_allclose(fabc(grid, b, c), expected)
     numpy.testing.assert_allclose(facb(grid, c, b), expected)
 
+
+def runtest_issue_10274(language, backend):
+    import tempfile
+    import os
+    expr = (a - b + c)**(13)
+    tmp = tempfile.mkdtemp()
+    f = autowrap(expr, language, backend, tempdir=tmp, helpers=('helper', a - b + c, (a, b, c)))
+    assert f(1, 1, 1) == 1
+
+    for file in os.listdir(tmp):
+        if file.startswith("wrapped_code_") and file.endswith(".c"):
+            fil = open(tmp + '/' + file)
+            assert fil.read() == ("/******************************************************************************\n"
+                         " *                    Code generated with sympy 0.7.7.dev                     *\n"
```

Comment:
This line will have to be changed with every release. It would be better if it was generalized. Either skip comparing this line or insert the version number here or something.